tests/rptest/tests/wasm_identity_test.py

```diff
@@ -210,7 +215,7 @@ def wasm_test_plan(self):
         ]
 
     @ignore  # https://github.com/vectorizedio/redpanda/issues/2514
-    @cluster(num_nodes=3)
+    @cluster(num_nodes=6)
```

Comment:
As I understand you also should fix num_nodes for WasmCreateTopicsTest.verify_materialized_topics_test

```
rptest.tests.wasm_topics_test.WasmCreateTopicsTest.verify_materialized_topics_test: Summary: InsufficientResourcesError('Not enough nodes available to allocate. linux nodes requested: 1. linux nodes available: 0')
```